### PR TITLE
changes login to use go API

### PR DESCRIPTION
**Pull request by dg4prez**
Modifies the ORT login to use the API rather than the old UI code.

### CODE REVIEW

File: traffic_ops/bin/traffic_ops_ort.pl [hunk 1]
```diff
@@ -1619,10 +1619,12 @@ sub get_cookie {
 		&sleep_rand($login_dispersion);
 	}
 
-	my $url = $to_host . "/login";
-	my $response = $lwp_conn->post( $url, [ 'u' => $u, 'p' => $p ], %headers );
+	my $url = $to_host . "/api/1.3/user/login";
+    my $json = '{"u" : "' . $u . '", "p" : "' . $p . '"}';
```
Comment (dangogh) [inline traffic_ops/bin/traffic_ops_ort.pl]: would be more readable using the qq operator:
```
my $json = qq/{ "u": "$u", "p": "$p"}/;
```

File: traffic_ops/bin/traffic_ops_ort.pl [hunk 2]
```diff
@@ -1619,10 +1619,12 @@ sub get_cookie {
 		&sleep_rand($login_dispersion);
 	}
 
-	my $url = $to_host . "/login";
-	my $response = $lwp_conn->post( $url, [ 'u' => $u, 'p' => $p ], %headers );
+	my $url = $to_host . "/api/1.3/user/login";
```
Comment (dangogh) [inline traffic_ops/bin/traffic_ops_ort.pl]: can you try to match previous indentation?

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/2197/
Test PASSed.

File: traffic_ops/bin/traffic_ops_ort.pl (hunk 1 above)
Comment (dg4prez) [reply traffic_ops/bin/traffic_ops_ort.pl]: thanks.  fixed.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/2198/
Test FAILed.

File: traffic_ops/bin/traffic_ops_ort.pl (hunk 2 above)
Comment (dg4prez) [reply traffic_ops/bin/traffic_ops_ort.pl]: My editor doesn't seem to agree with git about spaces.  should be fixed now.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/2199/
Test PASSed.

**Pull request merged**